view/frontend/web/js/fast-button-base.js

```diff
@@ -0,0 +1,17 @@
+define(['uiComponent'],
+    function(Component) {
+        'use strict';
+
+        return Component.extend({
+
+            fastConfig: function() {
+                return fastConfigFactory();
+            },
+            isFastDarkTheme: function() {
+                return self.fastConfig.getBtnTheme() === 'dark';
```

Comment:
I think this will throw an error `self is not defined`. you can do it as `this.fastConfig.getBtnTheme()` or declare self above like `var self = this`

Reply:
Good catch. I think this will affect some other places as well.